### PR TITLE
Update launch-dedicated.sh

### PR DESCRIPTION
**Pull request by Zimeon-**
Dedicated server logfile command https://github.com/OpenRA/OpenRA/issues/11289 example to launch-dedicated.sh.

### CODE REVIEW

**Review by pchote — COMMENTED** [review]
This seems like a reasonable thing to do, but i'd like to see the opinions of some of our server operators like @Baxxster and @jaZzKCS before merging.

File: launch-dedicated.sh [hunk 1]
```diff
@@ -18,5 +18,5 @@ while true; do
      mono --debug OpenRA.Server.exe Game.Mod=$Mod \
      Server.Name="$Name" Server.ListenPort=$ListenPort Server.ExternalPort=$ExternalPort \
      Server.AdvertiseOnline=$AdvertiseOnline \
-     Server.EnableSingleplayer=$EnableSingleplayer Server.Password=$Password
+     Server.EnableSingleplayer=$EnableSingleplayer Server.Password=$Password | tee /var/log/openra/server.log
```
Comment (pchote) [inline launch-dedicated.sh]: This won't work if the `/var/log/openra/` directory hasn't first been created.

**Comment by jaZzKCS** [comment]
Log files for every instance that append one integer up every time it loops would be greatly appreciated.
Another idea instead for having it auto-numbering up the files (log1, log2, etc) would be to make the server put date, mod, map and maybe players as well into the filename for easy sorting and making it easy for the server owner to look for a specific match/whether a specific match happened.

Ultimately I would LOVE to see log files being created for example like this..:

DATE(yyyymmdd)-TIME(hhmmss)-MOD-MAP(name or hash?)-PLAYER1-PLAYER2[...].log

20170315-224568-RA-Desert_Rats-OMNom-Daruke.log
20170517-135325-RA-Tournament_Island-pchote-Baxxster.log
20170520-155600-CNC-Hourglass-jaZz_KCS-Moz.log
20170611-205043-DK2-Plateaus-WhoCares-Killah_Mentor-Bot1-Bot2.log

This would make them be ordered chronologically and makes seeing what instance took place and who played what very easy. This could also lead to statistics being saved and being crunched for general usage data.

:+inf:

File: launch-dedicated.sh (hunk 1 above)
Comment (Zimeon-) [reply launch-dedicated.sh]: That's true, sorry about that. Perhaps add a folder named logs to base dir and have that as the default one?

**Comment by Baxxster** [comment]
I'm launching all my servers with the same bash-script, thus would make all instances go into the same log, and make a mess.
We need to be able to separate logfiles from instances, preferably on serverport as these will always be unique from instance to instance. So maybe "$ListenPort-$Date-server.log"

@jaZzKCS the log files you wanted there would need to be created on match start, and probably would require a lot more work than this simple workaround :p

**Comment by jaZzKCS** [comment]
> @jaZzKCS the log files you wanted there would need to be created on match start, and probably would require a lot more work than this simple workaround :p

Therefore I say ultimately. But having different logfiles for every different port together with a timestamp would be a great start! 

:+1:

**Comment by pchote** [comment]
What do we want to do here?

**Comment by pchote** [comment]
There isn't a clear direction forward here, so closing to help clear up the PR queue.

**Pull request closed without merging**